test/Dialect/TorchConversion/unpack-quant-tensor.mlir

```diff
@@ -0,0 +1,12 @@
+// RUN: torch-mlir-opt %s -torch-unpack-quant-tensor | FileCheck %s
+
+// CHECK-LABEL: func @forward
```

Comment:
I do need to ask you to actually verify the converted constants, given that the main thing the pass is doing is bit twiddling.

Reply:
Since you also changed the type legality code to include i4, I would like to see at least a variant of that.